xml/storage_docupdates.xml

```diff
@@ -104,6 +104,13 @@
         xlink:href="https://bugzilla.suse.com/show_bug.cgi?id=1099326"/>).
        </para>
       </listitem>
+      <listitem>
+       <para>
+        Ease configuration with <command>multipath -T</command> in
```

Comment:
To make configuration easier, use the ... command

Reply:
Thanks Dima! :+1: